::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Thu, 25 Feb 2016 16:01:18 +0100
Subject: [PATCH] Add FLASH_KERNEL_SKIP=true.

FIXME: Add nice description of the issue at hand.
---
 fuel_agent/utils/build.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/fuel_agent/utils/build.py b/fuel_agent/utils/build.py
index 2950ad8..74cbfd8 100644
--- a/fuel_agent/utils/build.py
+++ b/fuel_agent/utils/build.py
@@ -93,6 +93,7 @@ def set_apt_get_env():
     # NOTE(agordeev): disable any confirmations/questions from apt-get side
     os.environ['DEBIAN_FRONTEND'] = 'noninteractive'
     os.environ['DEBCONF_NONINTERACTIVE_SEEN'] = 'true'
+    os.environ['FLASH_KERNEL_SKIP'] = 'true'
     os.environ['LC_ALL'] = os.environ['LANG'] = os.environ['LANGUAGE'] = 'C'


